src/traits/is_callable.rs

```diff
@@ -30,6 +33,23 @@ pub trait IsCallable {
             .collect();
         quote!(#(#params),*)
     }
+
+    fn get_params_type_annotations(
```

Comment:
Would this read better if it was `get_param_type_annotations`?

Reply:
I decided on get_params because it was the type annotations for all of the params, not for one individual param